e2e-tests/playwright/e2e/plugins/github-actions/github-actions.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { expect, test } from "@playwright/test";
+import { UIhelper } from "../../../utils/ui-helper";
+import { Common } from "../../../utils/common";
+
+// Group the tests that test the GitHub Actions plugin functionality
+test.describe("Test GitHub Actions plugin functionality", () => {
+  // Reference component to test (should have GitHub Actions configured)
+  const componentWithGitHubActions = "Backstage Showcase";
+  
+  let uiHelper: UIhelper;
+  let common: Common;
+
+  test.beforeEach(async ({ page }) => {
+    // Increase default timeout to account for slower responses
+    test.setTimeout(180000); // 3 minutes
+    
+    common = new Common(page);
+    
+    // Login as GitHub user instead of guest
+    await common.loginAsGithubUser();
+    console.log("Logged in as GitHub user");
+    
+    uiHelper = new UIhelper(page);
+  });
+
+  test("Verify backstage-community-plugin-github-actions in Extensions page", async ({ page }) => {
+    // Navigate to Extensions page through Administration
+    await uiHelper.openSidebarButton("Administration");
+    await uiHelper.clickLink("Extensions");
+    console.log("Navigated to Extensions page");
+    
+    // Click on the Installed tab
+    await uiHelper.clickTab("Installed");
+    console.log("Clicked on Installed tab");
+    
+    // Wait for the table to load - using more reliable selector
+    await page.waitForSelector('table, div[role="grid"], [data-testid="plugins-table"]', { timeout: 30000 });
+    
+    // Search for the GitHub Actions plugin - use a shorter search term to improve matching
+    const pluginName = "backstage-community-plugin-github-actions";
+    const searchTerm = "github-actions"; // Use a shorter, more distinctive part of the name
+    await uiHelper.searchInputPlaceholder(searchTerm);
+    console.log(`Searching for plugin with term: ${searchTerm}`);
+    
+    // Wait for specific plugin row to appear after search
+    const pluginRow = page.locator(`tr:has-text("${pluginName}")`).first();
+    await expect(pluginRow).toBeVisible({ timeout: 10000 });
+    console.log("Found plugin row in table");
+    
+    // Instead of using verifyCellsInTable, we'll check more directly
+    const nameCell = pluginRow.locator('td').first();
+    await expect(nameCell).toContainText("github-actions");
+    
+    // Verify plugin status (Enabled and Preinstalled should both be "Yes")
+    const enabledCell = pluginRow.locator('td').nth(2);
```

Comment:
Can you please check UiHelper.ts? I think there's some methods which you can reuse for it, for example `verifyPluginRow`

Reply:
fixed